src/audio.py

```diff
@@ -10,7 +10,10 @@
 
 class Audio(QThread):
     _DEFAULT_VOLUME = 50
+    _DEFAULT_OITAVA = 0
     _MAX_VOLUME = 127
+    _MAX_OITAVA = 48
+    _MIN_OITAVA = -48
```

Comment:
Não vamos misturar português com inglês no código. O código está todo em inglês, então é melhor seguir o padrão.
```suggestion
    _DEFAULT_OCTAVE = 0
    _MAX_VOLUME = 127
    _MAX_OCTAVE = 48
    _MIN_OCTAVE = -48
```

Reply:
Este comentário vale para todas as ocorrências de palavras em português no código (com exceção de strings).